test/python/circuit/test_parameters.py

```diff
@@ -170,3 +172,70 @@ def test_parameter_name_conflicts_raises(self):
         qc.u1(theta1, 0)
 
         self.assertRaises(QiskitError, qc.u1, theta2, 0)
+
+    def test_bind_ryrz_vector(self):
+        qc = QuantumCircuit(4)
+        depth = 4
+        theta = ParameterVector('θ', length=len(qc.qubits) * depth * 2)
+        theta_iter = iter(theta)
+        for d in range(depth):
+            for q in qc.qubits:
+                qc.ry(next(theta_iter), q)
+                qc.rz(next(theta_iter), q)
+            for i, q in enumerate(qc.qubits[:-1]):
+                qc.cx(qc.qubits[i], qc.qubits[i+1])
+            qc.barrier()
+        theta_vals = np.random.random(len(theta)) * np.pi
```

Comment:
Can you fix this value? Test should be deterministic #205 :)

Reply:
Fixed